### PR TITLE
Adding pyophidia recipe

### PR DESCRIPTION
**Pull request by doutriaux1**

### CODE REVIEW

File: recipes/pyophidia/build.sh [hunk 1]
```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env python
```
Comment (synapticarbors) [inline recipes/pyophidia/build.sh]: Can we drop `build.sh` in favor of:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L29

File: recipes/pyophidia/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    run:
+        - python
+
+test:
+    imports:
+        - PyOphidia
+
+about:
+    home:  http://github.com/OphidiaBigData/PyOphidia
+    license:  GPLv3
```
Comment (synapticarbors) [inline recipes/pyophidia/meta.yaml]: `GPL-3.0`

File: recipes/pyophidia/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,35 @@
+package:
+    name: pyophidia
+    version: "1.4.0"
+
+source:
+    url: git://github.com/OphidiaBigData/PyOphidia/archives/v1.4.0.tar.gz
+    sha256: b276b11d8e72396f00a95eeec0086df75e6fe6980106eb2fab68d237c665a1f7
+
+build:
```
Comment (synapticarbors) [inline recipes/pyophidia/meta.yaml]: It looks like this package might be a candidate for using `noarch`:

https://conda-forge.org/docs/meta.html#building-noarch-packages

File: recipes/pyophidia/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+about:
+    home:  http://github.com/OphidiaBigData/PyOphidia
+    license:  GPLv3
+    licence_file: LICENSE
+    summary: PyOphidia is a Python package for interacting with the Ophidia framework
+
+extra:
+    recipe-maintainers:
+        - doutriaux1
+        - eldoo
```
Comment (synapticarbors) [inline recipes/pyophidia/meta.yaml]: @eldoo, @minomicetto, @aledanca, @sandrofiore -- Are you ok being added as co-maintainers of the `PyOphidia` package on conda-forge?

File: recipes/pyophidia/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,35 @@
+package:
+    name: pyophidia
+    version: "1.4.0"
+
+source:
+    url: git://github.com/OphidiaBigData/PyOphidia/archives/v1.4.0.tar.gz
```
Comment (synapticarbors) [inline recipes/pyophidia/meta.yaml]: this should be an `http` url not a `git` on.  Can you just use the following canonical url:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L18

it looks like it should be on pypi:
https://pypi.org/project/PyOphidia/#files

Note, you might consider using the jinja templating features as in:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L4-L19

just be aware that the name should be `PyOphidia` not `pyophidia` given the naming on pypi.

File: recipes/pyophidia/meta.yaml (hunk 3 above)
Comment (doutriaux1) [reply recipes/pyophidia/meta.yaml]: I though conda-forge couldn't do `noarch`, i'll look into this.

**Comment by synapticarbors** [comment]
I wonder if it's getting the correct file. The link on github is:

https://github.com/OphidiaBigData/PyOphidia/archive/v1.4.0.tar.gz

whereas you have `archives` (plural). Maybe that's the issue?

File: recipes/pyophidia/meta.yaml (hunk 4 above)
Comment (sandrofiore) [reply recipes/pyophidia/meta.yaml]: Yes, it is fine with me.

Comment (eldoo) [reply recipes/pyophidia/meta.yaml]: Yes, it's ok with me.

Comment (minomicetto) [reply recipes/pyophidia/meta.yaml]: Yes, of course

Comment (aledanca) [reply recipes/pyophidia/meta.yaml]: Yes, sure.

**Comment by synapticarbors** [comment]
Everything looks good, but CircleCI isn't running, which is important for the `noarch`.

@conda-forge/staged-recipes -- any thoughts on how to trigger the necessary build?

**Comment by doutriaux1** [comment]
@synapticarbors any progress on this? It's all green? Should I submit another PR in order to try to trigger CircleCI? @conda-forge/staged-recipes ping too.

**Comment by scopatz** [comment]
Let's try closing and reopening

**Comment by scopatz** [comment]
Seemed to work!

**Comment by synapticarbors** [comment]
Still don't see CircleCI, which was the original issue that was holding this up.

**Comment by scopatz** [comment]
Oh whoops, I was looking for travis!

**Comment by scopatz** [comment]
Yeah, maybe try a new PR

**Comment by jakirkham** [comment]
Guessing the issue is it is coming from the `UV-CDAT` organization. I think either CircleCI needs to be enabled for the organization (not the repo) or it needs to be submitted from a user account that has CircleCI enabled.

**Comment by doutriaux1** [comment]
hum.... will try be this org has had MANY successful PR with Circle-ci in the past. Will take a look though, it's worth a try.

**Comment by jakirkham** [comment]
It's possible CircleCI has changed something that will need to be addressed by some org setting change or refresh.

**Comment by doutriaux1** [comment]
ok went to setting and truned it on, then submitted a new PR from nbew branch (at https://github.com/conda-forge/staged-recipes/pull/4299)
circleci is now in and passing. I think we can merge. Thanks for all of your help.

**Comment by dougalsutherland** [comment]
Travis and AppVeyor passed on the previous basically-identical commit, so merging without waiting for them. I did add a `license_family: GPL`, though.

**Pull request merged**